File: docs/examples/cpp/src/lifecycle_policy_remove.cc
#include <cassert>
#include <reduct/client.h>

using reduct::Error;
using reduct::IClient;

int main() {
  auto client = IClient::Build("http://127.0.0.1:8383", {.api_token = "my-token"});

  auto err = client->DeleteLifecycle("lifecycle-to-remove");
  assert(err == Error::kOk);
}

File: versioned_docs/version-1.21.x/examples/go/src/lifecycle_policy_remove.go
package main

import (
	"context"

	reduct "github.com/reductstore/reduct-go"
)

func main() {
	client := reduct.NewClient("http://localhost:8383", reduct.ClientOptions{APIToken: "my-token"})

	if err := client.RemoveLifecycle(context.Background(), "lifecycle-to-remove"); err != nil {
		panic(err)
	}
}
